dm/syncer/syncer.go

```diff
@@ -217,6 +219,7 @@ type Syncer struct {
 	}
 
 	addJobFunc func(*job) error
+	seq        int64
```

Comment:
do we need reset `seq` in `syncer.reset()`

Reply:
in fact it's not possible for int64 to overflow